provisioning/baremetal_pod.go

```diff
@@ -258,28 +257,35 @@ func newMetal3InitContainers(info *ProvisioningInfo) []corev1.Container {
 	}
 
 	initContainers = append(initContainers, createInitContainerMachineOsDownloader(info, true))
+	initContainers = append(initContainers, createInitContainerCoreOSIPA(info))
 
 	return injectProxyAndCA(initContainers, info.Proxy)
 }
 
-func createInitContainerIpaDownloader(images *Images) corev1.Container {
+func createInitContainerCoreOSIPA(info *ProvisioningInfo) corev1.Container {
+	config := &info.ProvConfig.Spec
 	initContainer := corev1.Container{
-		Name:            "metal3-ipa-downloader",
-		Image:           images.IpaDownloader,
-		Command:         []string{"/usr/local/bin/get-resource.sh"},
+		Name:            "metal3-configure-coreos-ipa",
+		Image:           info.Images.Ironic,
+		Command:         []string{"/bin/configure-coreos-ipa"},
 		ImagePullPolicy: "IfNotPresent",
 		SecurityContext: &corev1.SecurityContext{
 			Privileged: pointer.BoolPtr(true),
 		},
 		VolumeMounts: []corev1.VolumeMount{imageVolumeMount},
```

Comment:
You may also need to mount the TLS volume so that it knows that TLS is used.

Reply:
Ref: https://github.com/openshift/ironic-image/blob/master/scripts/configure-coreos-ipa#L15